euclidean_gcd/euclidean_gcd.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+
+/*
+* first --> First number
+* second --> Second number
+* There are two implementations: Recursive (gcd_recursive) and Non-Recursive (gcd_iterative)
+*/
+
+int gcd_recursive(int first, int second) {
+        if(second == 0)
+            return first;                //first becomes gcd if second becomes zero
+        else
+            return gcd_recursive(second, (first % second));
+}
+
+int gcd_iterative(int first, int second) {
```

Comment:
But I had to differentiate the recursive and iterative implementation thats why I named them gcd_recursive and gcd_iterative.
Shall I make it euclidean_gcd_recursive and euclidean_gcd_iterative?